x-pack/plugins/observability_ai_assistant/public/components/action_menu_item/action_menu_item.tsx

```diff
@@ -86,19 +138,22 @@ export function ObservabilityAIAssistantActionMenuItem() {
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiHeaderLink>
+
       {chatService.value ? (
         <ObservabilityAIAssistantChatServiceProvider value={chatService.value}>
           <ChatFlyout
+            initialConversationId={flyoutState.conversationId}
             initialTitle=""
-            initialMessages={initialMessages}
+            initialMessages={[]}
```

Comment:
Fair!

Reply:
Instead of the previous approach:
```ts
  const initialMessages = useMemo(() => [], []);
```

You can define it outside of `ObservabilityAIAssistantActionMenuItem`:
```ts
  const initialMessages = [];

  export function ObservabilityAIAssistantActionMenuItem() {
```